examples/c/ex_all.c

```diff
@@ -1373,5 +1373,23 @@ main(int argc, char *argv[])
         (void)crc32c;
     }
 
+    {
+        const char *buffer = "some other larger string";
+        size_t chunk1, chunk2, chunk3;
+        /*! [Checksum a large buffer in chunks] */
+        uint32_t crc32c, (*func)(const void *, size_t);
+        uint32_t (*func_with_seed)(uint32_t, const void *, size_t);
```

Comment:
Add a blank line here.

Reply:
Fixed in https://github.com/wiredtiger/wiredtiger/pull/9900/commits/895b8d0771f065c19e678d9125a9684fcc9bd02c